facebook/react-native
# migrate MyNativeView to Kotlin

## Summary:

This PR converts MyNativeView into Kotlin as requested in https://github.com/facebook/react-native/issues/38825 .

## Changelog:


[INTERNAL] [CHANGED] - Migrate MyNativeView to Kotlin


## Test Plan:

1. run `yarn android`
2. Check whether RN Tester runs as expected
